composer.lock

```diff
@@ -1907,6 +1549,79 @@
             ],
             "time": "2022-06-09T09:09:00+00:00"
         },
+        {
+            "name": "promphp/prometheus_client_php",
+            "version": "v2.6.1",
+            "source": {
+                "type": "git",
+                "url": "https://github.com/PromPHP/prometheus_client_php.git",
+                "reference": "918b9962f0d5f2ecf401aa99b334a1620b6aeffe"
+            },
+            "dist": {
+                "type": "zip",
+                "url": "https://api.github.com/repos/PromPHP/prometheus_client_php/zipball/918b9962f0d5f2ecf401aa99b334a1620b6aeffe",
+                "reference": "918b9962f0d5f2ecf401aa99b334a1620b6aeffe",
+                "shasum": "",
+                "mirrors": [
+                    {
+                        "url": "https://mirrors.aliyun.com/composer/dists/%package%/%reference%.%type%",
+                        "preferred": true
+                    }
+                ]
```

Comment:
Why this? 🙂

Reply:
Oh,I used a mirror site because of the network environment and I forgot to remove it